package.json

```diff
@@ -37,6 +37,7 @@
     "grunt-contrib-uglify": "0.8.0",
     "grunt-mocha-test": "0.12.7",
     "grunt-shell-spawn": "0.3.1",
+    "mocha": ">= 1.20.0",
```

Comment:
@jh-bate @jebeck I'm fine with choosing a specific version.  We can choose the latest one it installs right now (2.2.5).  The reason I added it this way was because this specific dependency was being pulled in implicitly (rather than explicitly) from the inclusion of grunt-mocha-test (and implicit peer dependencies will be deprecated in the future).